qa/pull-tester/rpc-tests.py

```diff
@@ -184,26 +176,39 @@
     'replace-by-fee.py',
 ]
 
+ALL_SCRIPTS = BASE_SCRIPTS + ZMQ_SCRIPTS + EXTENDED_SCRIPTS
 
 def runtests():
-    test_list = []
-    if '-extended' in opts:
-        test_list = testScripts + testScriptsExt
-    elif len(opts) == 0 or (len(opts) == 1 and "-win" in opts):
-        test_list = testScripts
+    # Build list of tests
+    if len(tests) != 0:
```

Comment:
Should avoid if len in python, https://www.python.org/dev/peps/pep-0008/

```
Yes: if not seq:
     if seq:

No: if len(seq):
    if not len(seq):
```

More occurrences a few places below, too.